### DISCUSSION TITLE
How to set transformer zindex ?

### DISCUSSION THREAD

**Discussion by lanyusan** [konvajs/konva#1224] [Q&A] [answered]
Hi,

I am working on a prototype with Konva now.

I have a use case that needs to bring some shape to front by setting zindex then select it to show a transformer box. But the transformer outline is still behind the shape originally on top.

Is there a way to get the bonding outline / anchors  of the transformer and set their zindex?

Thanks.

**Comment by lavrton** [comment, marked as answer]
Transformer is the same node as any other. It is just a `Konva.Group` with some special abilities.
So, you can control its zIndex as for another node.

**Reply by lanyusan** [reply]
Yes it works. It is just zIndex is not mentioned in Transformer class document. Maybe update it?
